homeassistant/components/smarty/fan.py

```diff
@@ -0,0 +1,133 @@
+"""
+Platform to control a Salda Smarty XP/XV ventilation unit.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/fan.smarty/
+"""
+import logging
+
+from homeassistant.core import callback
+from homeassistant.components.fan import (
+    SPEED_HIGH, SPEED_LOW, SPEED_MEDIUM, SPEED_OFF, SUPPORT_SET_SPEED,
+    FanEntity)
+from homeassistant.components.smarty import (
```

Comment:
Use relative imports when importing from the smarty package. This will allow custom component override.
```py
from . import (...)
```